dotcom-rendering/src/lib/content.d.ts

```diff
@@ -286,6 +286,11 @@ interface MultiImageBlockElement {
 	role?: RoleType;
 }
 
+interface NewsletterSignupBlockElement {
+	_type: 'model.dotcomrendering.pageElements.NewsletterSignupBlockElement';
```

Comment:
I wonder if we will need an `elementId` here? This is a [property that gets added](https://github.com/guardian/frontend/blob/43fd3eed9641e23211dc8e1b743120eca362040f/common/app/model/dotcomrendering/PageElement-Identifiers.md) by Frontend at some point so that we can have a unique id